### PR TITLE
Fixed logtools command output

### PR DESCRIPTION
**Pull request by braden6521**
- Fixed Issue #14 where multiple outputs of the same level were printed in console. 
- Pep8-ified the document and cleaned up where necessary.

### CODE REVIEW

File: opencsp/common/lib/tool/log_tools.py [hunk 1]
```diff
@@ -153,51 +149,22 @@ def multiprocessing_logger(log_dir_body_ext=None, level=log.INFO):
...
-    h1.addFilter(lambda record: record.levelno <= level)
+    h1.addFilter(lambda record: record.levelno >= level)
     h1.setFormatter(formatter)
-    logger.addHandler(h1)
-
-    # stderr log
-    h2 = log.StreamHandler(sys.stderr)
-    h2.setLevel(log.WARNING)
-    h2.setFormatter(formatter)
-    logger.addHandler(h2)
```
Comment (braden6521) [inline opencsp/common/lib/tool/log_tools.py:174]: I believe the error was caused by two things. 
1. This inequality was backwards
2. Having both stderr and stdout 

So I removed stderr and flipped the inequality.

Comment (bbean23) [reply opencsp/common/lib/tool/log_tools.py:174]: I disagree about removing stderr (see below for my reasoning why).

Looking at this code again, here's what I propose:

```python
    # stdout: log everything between level and warning
    h1 = log.StreamHandler(sys.stdout)
    h1.setLevel(level)
    h1.addFilter(lambda record: record.levelno < log.WARNING)
    h1.setFormatter(formatter)
    logger.addHandler(h1)

    # stderr: log everything warning and greater (warning, error, critical)
    h2 = log.StreamHandler(sys.stderr)
    h2.setLevel(log.WARNING)
    h2.setFormatter(formatter)
    logger.addHandler(h2)
```

Comment (bbean23) [reply opencsp/common/lib/tool/log_tools.py:174]: Reasons to keep stderr:
- Often times application integrators will limit output to only stderr as a first-pass filter for important information.
- Stderr also is implemented differently on most OSs. It isn't buffered, meaning it is more likely that the critical part of the log will be printed in the case that the program crashes halfway through printing the line.

**Review by bbean23 — COMMENTED** [review]
It looks like you found the culprit! Although I'm not convinced this is the right fix, I'm still open to the idea. What do you think of my proposed solution?

File: opencsp/common/lib/tool/log_tools.py [hunk 2]
```diff
@@ -255,17 +222,17 @@ def warn(*vargs, **kwargs):
         lt.logger(home_dir() + 'current.log')
         lt.warn('In my_function(), Plot x label is empty.')
     """
-    if global_multiprocessing_logger != None:
-        global_multiprocessing_logger.warn(*vargs, **kwargs)
+    if global_multiprocessing_logger is not None:
+        global_multiprocessing_logger.warning(*vargs, **kwargs)
```
[the remark this answers is not in the document]
Comment (braden6521) [reply opencsp/common/lib/tool/log_tools.py:232]: Yeah! I would not have known either if the "pylint" VSCode plugin didn't tell me 😉

File: opencsp/common/lib/tool/log_tools.py (hunk 1 above)
Comment (braden6521) [reply opencsp/common/lib/tool/log_tools.py]: Oh, that makes a lot of sense!! Thanks for the code proposal, I think that's 100% correct. It seems to work now.

**Comment by braden6521** [comment]
@bbean23, I implemented your suggestion. Seems to work correctly. Thanks for your expertise!

**Pull request merged**